app/src/main/res/layout/fragment_public_restaurant_info.xml

```diff
@@ -0,0 +1,172 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/im_public_restaurant_address"
+        android:layout_width="14dp"
+        android:layout_height="14dp"
+        android:layout_marginStart="13dp"
+        android:layout_marginTop="20dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_info_address" />
+
+    <TextView
+        android:id="@+id/tv_public_restaurant_address"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="11dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="64dp"
+        android:textColor="@color/brownish_grey"
+        android:textSize="12sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/im_public_restaurant_address"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Ground, First and Second Floor, Orient Bestech Business
+Towers, National Highway 8, Block A, Sector 34,
+Gurugram, Haryana 122004
+" />
+
+    <ImageView
+        android:id="@+id/im_public_restaurant_address_btn"
+        android:layout_width="12dp"
+        android:layout_height="12dp"
+        app:layout_constraintBottom_toBottomOf="@+id/tv_public_restaurant_address"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/tv_public_restaurant_address"
+        app:layout_constraintTop_toTopOf="@+id/tv_public_restaurant_address"
+        app:srcCompat="@drawable/ic_info_right" />
+
+    <View
+        android:id="@+id/div"
+        android:layout_width="0dp"
+        android:layout_height="0.3dp"
+        android:layout_marginStart="13dp"
+        android:layout_marginTop="14dp"
+        android:layout_marginEnd="13dp"
+        android:background="?android:attr/listDivider"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/tv_public_restaurant_address" />
+
+    <ImageView
+        android:id="@+id/im_public_restaurant_number"
+        android:layout_width="12dp"
+        android:layout_height="12dp"
+        android:layout_marginStart="13dp"
+        android:layout_marginTop="15dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/div"
+        app:srcCompat="@drawable/ic_info_call" />
+
+    <TextView
+        android:id="@+id/tv_public_restaurant_phone"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="11dp"
+        android:layout_marginTop="15dp"
+        android:layout_marginEnd="64dp"
+        android:textColor="@color/brownish_grey"
```

Comment:
Color, size and font... all are set in one of the various Text style. So please use that for maximum readability.